qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -38,7 +40,25 @@
 
 
 class IBMQFactory:
-    """Factory and account manager for IBM Quantum Experience."""
+    """Factory and account manager for IBM Quantum Experience.
+
```

Comment:
I think this block comment can use more words before jumping into `save_account()`.

Reply:
What I meant was that since this block comment is meant to describe the _the entire IBMQFactory class_, it should contain more information than just how to save a default provider. Like it should explain what this class is used for and the terminologies (token, provider, configuration file, etc). Perhaps this can go into a different doc PR.
<br>
Note that my original comment regarding the docstring was
>Perhaps a nice, concise description, of how to use IBMQFactory to manage accounts can go into IBMQFactory's docstring. 
<br>
Also while it doesn't hurt to give users more information, I don't think it's necessary to describe in details how to specify a default provider.